_pages/home.md

```diff
@@ -5,30 +5,26 @@ excerpt: "WhiteMech at Sapienza University."
 sitemap: false
 permalink: /
 ---
-
-WhiteMech aims at developing the science and the tools for mechanisms that are able to program themselves, automatically tailor their behavior so as to achieve desired goals, maintain themselves within safe boundaries in a changing environment, and follow regulations and conventions that evolve over time. Crucially, empowering mechanisms with self-programming carries significant risks and therefore we must be able to balance power with safety. For this reason WhiteMech intends to realize mechanisms that are white-box, that is, whose behavior is at any moment fully analyzable and comprehensible in human terms, and guarded by human oversight.
-
-
-  <!-- <img src="{{ site.url }}{{ site.baseurl }}/images/logopic/Logo_WhiteMech.svg" style="width: 100px"> -->
-
-### Context
+<!--
+### Context.
+<p class="text-justify">
 We are witnessing an increasing availability of mechanisms that offer form of programmability. These mechanisms are currently being revolutionized by advancements in sensing (vision, language understanding) and actuation components (autonomous mobile manipulators, automated storage and retrieval systems). However, such mechanisms are held back by the fact that their logic is still based on hard-wired rules encoded in hand-crafted programs.
-
-### How
+</p>
+### How.
+<p class="text-justify">
 Recent discoveries in Knowledge Representation, Reasoning about Actions and Planning, and Reinforcement Learning in Artificial Intelligence, and their connections to Verification and Synthesis in Formal Methods, and Data-Aware Processes in Databases, chart an novel path to produce a breakthrough.
-
-### Driving Applications
+</p>
+### Driving Applications.
+<p class="text-justify">
 WhiteMech will ground its scientific results upon three driving applications: smart manufacturing (Industry 4.0), smart spaces (IoT) and business process management systems (BPM).
-
+</p>
 We are located at Sapienza University.
 
 **Positions available as Senior Postdoc, Junior Postdocs and PhD students.** [(more info)]({{ site.url }}{{ site.baseurl }}/vacancies) **!**
+-->
 
-
-We are grateful for funding from European Union, [ERC Advanced Grant](https://erc.europa.eu/funding/advanced-grants).
-
+We are grateful for funding from the European Union with an [ERC Advanced Grant](https://erc.europa.eu/funding/advanced-grants).
 <figure class="fourth">
-  <img src="{{ site.url }}{{ site.baseurl }}/images/logopic/Logo_WhiteMech.svg" style="width: 110px">
+  <img src="{{ site.url }}{{ site.baseurl }}/images/logopic/logo-erc.svg" style="width: 25%">
```

Comment:
I can't see this picture, maybe it needs to be renamed?

Reply:
no, it has to be added and committed